media/js/mozorg/home/takeover.js

```diff
@@ -0,0 +1,119 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+$(function () {
+    'use strict';
+
+    // indexOf on Array is undefined in versions of IE lte8
+    var oldIE = typeof Array.prototype.indexOf !== 'function';
```

Comment:
Alternatively, we could rename it to `cutsTheMustard`? And feature detect `indexOf` and `sessionStorage`?